Feedback/templates/Feedback/teacher_check.html

```diff
@@ -4,45 +4,63 @@
+			  <datalist id="teacher_dept"class="check-datalist">
+				<option selected="disabled" value="none"></option>
+				  {% for element in teacher %}
+					<option value="{{ element.teacher_dept }}"></option>
+				  {%endfor%}
+			  </datalist>
+			</input>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+		  Faculty:-
+			<input list="teacher_name" name='dropdown1' autocomplete="off">
+			  <datalist id="teacher_name"class="check-datalist">
+				<option selected="disabled" value="none"></option>
+				  {% for element in teacher %}
+					<option value="{{ element.teacher_name }}"></option>
+				  {%endfor%}
+			  </datalist>
+			</input>
+		</h3>
+		</div><br><br>	
+		<button class="btn btn-primary check-button"name="submit" type="submit" name = 'Submit'  >Submit</button><br><br>
+		
+		{%if Flag == 1%}
+			<div class="detail">
+				<center><h1 class="check-detail-center-h1">Details of {{faculty}}</h1><p class="check-detail-center-p">({{department}})</p></center>
 
-<!--	<nav class="navbar navbar-expand-lg bg-secondary fixed-top text-uppercase" style="margin-top: 10px;">	-->
-	<h1 align="center"class="check-h1">Feedback Details</h1><br><br>
-	<form action="{%url 'teacher_check'%}" method="POST" class="check-form">
-			{%csrf_token%}
-	<div class="form">		
-	<h3>
-		Department:-
-		<input list="teacher_dept" name ="dropdown" autocomplete="off">
-				<datalist id="teacher_dept"class="check-datalist">
-						<option selected="disabled" value="none"></option>
-						{% for element in teacher %}<option value="{{ element.teacher_dept }}"></option>{%endfor%}
-				</datalist>
-		</input>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-		Faculty:-
-		<input list="teacher_name" name='dropdown1' autocomplete="off">
-				<datalist id="teacher_name"class="check-datalist">
-						<option selected="disabled" value="none"></option>
-						{% for element in teacher %}<option value="{{ element.teacher_name }}"></option>{%endfor%}
-				</datalist>
-		</input>
-	</h3>
-</div><br><br>
-	<button class="btn btn-primary check-button"name="submit" type="submit" name = 'Submit'  >Submit</button><br><br>
-	{%if Flag == 1%}
-	<div class="detail">
-			<center><h1 class="check-detail-center-h1">Details of {{faculty}}</h1><p class="check-detail-center-p">({{department}})</p></center>
-
-			<table>
-				 <thead>
-				<th >Section</th><th>Punctuality</th><th>Subject_knowledge</th><th>Behaviour</th><th>Method_of_teaching</th><th>Audibility</th><th>Syllabus_coverage</th></thead>
-				{%for F in Feedback%}
-					{%if F.teacher.teacher_name == faculty and F.teacher.teacher_dept == department%}
-					<tr><td >{{F.section}}</td><td>{{F.Punctuality}}</td><td>{{F.Subject_knowledge}}</td><td>{{F.Behaviour}}</td><td>{{F.Method_of_teaching}}</td><td>{{F.Audibility}}</td><td>{{F.Syllabus_coverage}}</td></tr>
-					{%endif%}
-				{%endfor%}
-			</table>
-	</div>			
+				<table>
+					 <thead>
+						<th >Section</th>

```

Comment:
Why have you given this much space for identation here? It should be consistent throughout the file.
if you are using two spaces for identation then use two only for every tag.

Reply:
It is showing consistent through all file in my laptop.